stream_alert_cli/__init__.py

```diff
@@ -1,2 +1,2 @@
 """Define the version of the CLI."""
-__version__ = '1.4.0'
```

Comment:
should this just point to the main lib version?

Reply:
I wasn't sure if there was a reason the cli was versioned separately. should we remove this?

Reply:
yeah let's just keep them consistent for now.